### PR TITLE
Add:Notification model #95

### PR DESCRIPTION
**Pull request by jaypajji4444**
Created the Notification model  with dynamic reference for event_id based on model field.

### CODE REVIEW

File: models/notification.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose');
+
+const NotificationSchema =new mongoose.Schema({
+    fromUser:{
+        type: mongoose.Schema.Types.ObjectId,
+        ref: 'User'
+    },
+    toUser:{
```
Comment (vixrant) [inline models/notification.js:8]: This isn't required, can be figured out from event_id doc.

File: models/notification.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose');
+
+const NotificationSchema =new mongoose.Schema({
+    fromUser:{
```
Comment (vixrant) [inline models/notification.js:4]: Rename to user.

File: models/notification.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    },
+    event_id:{
+        type:mongoose.Schema.Types.ObjectId,
+        required:true,
+        refPath: 'model',
+    },
+    model:{
+        type: String,
+        required: true,
+        enum: ['Forum', 'chat']
```
Comment (vixrant) [inline models/notification.js:32]: 'chat' -> 'Chat'

Comment (jaypajji4444) [reply models/notification.js:32]: ohk ..Correcting it...

**Pull request merged**